examples/functions/simpleNetwork/Netex_SimpleNetwork_1.xml

```diff
@@ -606,10 +606,24 @@ v1.1 revised ids
 								<PointOnRoute version="any" id="mybus:RPIS_001o" order="0001">
 									<ScheduledStopPointRef version="032" ref="hde:SSP0001A"/>
 									<OnwardRouteLinkRef version="any" ref="hde:hde:RL_001o@SSP0001A+RteP021"/>
+									<routeInstructions>
+										<RouteInstruction id="ri1" version="any">
+											<Instruction>false</Instruction>
```

Comment:
We probably need to be careful about these autogenerated example (I recognise the "false" from XMLSpy) ... her the valuses for <Instruction> and <Roadname>, below, are a bit misleading

Reply:
reduced it